specifyweb/frontend/js_src/lib/components/SchemaConfig/Tables.tsx

```diff
@@ -75,18 +75,33 @@ export function TableList({
 }: {
   readonly cacheKey: CacheKey;
   readonly getAction: (table: SpecifyTable) => string | (() => void);
-  readonly filter?: (table: SpecifyTable) => boolean;
+  readonly filter?: (showHiddenTables: boolean, table: SpecifyTable) => boolean;
```

Comment:
just a friendly reminder:
the TableList component is used in multiple places and on changes you have to consider impact in app places